### PR TITLE
Add mention of membership roles to contrib guide

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

**Comment by castrojo** [comment]
Fixes #1826

File: contributors/guide/README.md [hunk 1]
```diff
@@ -55,11 +55,12 @@ Please make sure to read and observe our [Code of Conduct](https://github.com/cn
...
 If you haven’t set up your environment, please find resources [here](/contributors/devel).
 
-## Community Expectations
+## Community Expectations and Roles
 
 Kubernetes is a community project. Consequently, it is wholly dependent on its community to provide a productive, friendly and collaborative environment.
 
-Read and review the [Community Expectations](community-expectations.md) for an understand of code and review expectations. 
+- Read and review the [Community Expectations](community-expectations.md) for an understand of code and review expectations. 
+- See [Community Membership](/community-membership.md) for a list the various responsibilities of contributor roles.
```
Comment (guineveresaenger) [inline contributors/guide/README.md]: Could we add some new contributor context for this? New contributors aren't members, after all.

Maybe: "We encourage regular contributors to grow into community membership roles; please see a list of the various responsibilities here".

File: contributors/guide/README.md [hunk 2]
```diff
@@ -55,11 +55,12 @@ Please make sure to read and observe our [Code of Conduct](https://github.com/cn
 
 If you haven’t set up your environment, please find resources [here](/contributors/devel).
 
-## Community Expectations
+## Community Expectations and Roles
```
Comment (cblecker) [inline contributors/guide/README.md]: If we're changing this title, we need to fix the TOC on line 22

File: contributors/guide/README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Welcome to Kubernetes! This document is the single source of truth for how to co
     -   [Code of Conduct](#code-of-conduct)
     -   [Setting up your development
         environment](#setting-up-your-development-environment)
-    -   [Community Expectations](#community-expectations)
+    -   [Community Expectations and Roles](#community-expectations)
```
Comment (tpepper) [inline contributors/guide/README.md]: I think you'll need it to be:

```[Community Expectations and Roles](#community-expectations-and-roles)```

Comment (tpepper) [reply contributors/guide/README.md]: affirmative, just tested.  You want ^^

File: contributors/guide/README.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Welcome to Kubernetes! This document is the single source of truth for how to co
...
         -   [Thanks](#thanks)
```
Comment (tpepper) [inline contributors/guide/README.md]: Looks like this one's out of place now too.  ToC needs a scrub.

**Pull request merged**